### DISCUSSION TITLE
出现bug了吗？时间错误。

### DISCUSSION THREAD

**Discussion by luoliguang** [Quorafind/Obsidian-Thino#642] [Q&A 问答]
![image](https://github.com/Quorafind/Obsidian-Thino/assets/74090098/de9574c4-d413-486e-8f5a-1dcc6f2b37f1)

**Comment by WANG-Yuuu** [comment]
同问

**Reply by EndlessCCL** [reply]
同问

**Comment by juestchaos** [comment]
提交更详细的产生原因，只看截图，无法知道这个时间是怎么错了
已知1，用户自己调整时间
已知2，用户保存日记路径发生了异常
